exec/java-exec/src/test/java/org/apache/drill/exec/sql/TestAnalyze.java

```diff
@@ -492,7 +492,7 @@ public void testHistogramWithDataTypes1() throws Exception {
           .match();
 
       query = "select 1 from dfs.tmp.employee1 where store_id < 15";
-      String[] expectedPlan2 = {"Filter\\(condition.*\\).*rowcount = 676.*,.*",
+      String[] expectedPlan2 = {"Filter\\(condition.*\\).*rowcount = 699.*,.*",
               "Scan.*columns=\\[`store_id`\\].*rowcount = 1128.0.*"};
```

Comment:
Do you know why rowcount is changed?

Reply:
@vdiravka yes, please see the comments on this PR from @tdunning.  The t-digest data structure changed a bit in 3.3 and estimates like this rowcount = 676 are expected to be a bit more accurate.